graphics/include/ignition/common/ColladaExporter.hh

```diff
@@ -32,6 +33,22 @@ namespace ignition
 {
   namespace common
   {
+    /// \brief This struct contains light data specifically for collada export
+    /// Defaults set based on collada 1.4 specifications
+    struct ColladaLight
+    {
+      std::string name;  // name of the light
```

Comment:
Please document with `\brief` above the members, inline comments are discouraged

Reply:
documented ColladaLight members 1b7602b